### PR TITLE
minor fix-change GetUIDomain to return hostname  or localhost

### PR DESCRIPTION
**Pull request by sunitakawane**
# Description
Summarize the change.

Fixes #<issue-number> (link all the GitHub issues this addresses)

# Testing
Describe how you tested this code. How can the reviewers reproduce your tests?

# Checklist:
Before marking your pull request as ready for review, complete the following.

- [ ] I have self-reviewed this PR.
- [ ] I have removed all print-debugging and commented-out code that should not be merged.
- [ ] I have added sufficient comments in my code, particularly in hard-to-understand areas.
- [ ] I have formatted the code, if applicable. For Go, I have run `goimports -w .`.

### CODE REVIEW

**Review by AndrewHess — COMMENTED** [review]
The change looks good, but it doesn't do what the PR title says it does

File: pkg/config/config.go [hunk 1]
```diff
@@ -238,7 +239,12 @@ func SetEmailConfig(smtpHost string, smtpPort int, senderEmail string, gmailAppP
 }
 
 func GetUIDomain() string {
-	return GetQueryHostname()
+	hostname := GetQueryHostname()
+	parsedUrl, err := url.Parse(hostname)
+	if err != nil {
+		log.Fatalf("GetUIDomain: Failed to parse QueryHostname: %v, err: %v", hostname, err)
```
Comment (AndrewHess) [inline pkg/config/config.go:245]: Are you sure we want to crash if we get an error here? This function could feasibly get called at any point, not just at startup

File: pkg/config/config.go [hunk 2]
```diff
@@ -238,7 +239,12 @@ func SetEmailConfig(smtpHost string, smtpPort int, senderEmail string, gmailAppP
...
+	}
+	return parsedUrl.Host
```
[the remark this answers is not in the document]
Comment (sunitakawane) [reply pkg/config/config.go:247]: It is being set to locahost:port in config.go https://github.com/siglens/siglens/blob/bf755876ade29643b4edbf47e04ba3cd94ff0e2d/pkg/config/config.go#L789

File: pkg/config/config.go (hunk 1 above)
Comment (sunitakawane) [reply pkg/config/config.go:245]: I changed this to Errorf and setting it to localhost in case of error

**Pull request merged**